client/src/components/ProductListing/ProductListing.spec.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react'
+import ProductListing from './ProductListing'
+
+describe('ProductListing', () => {
+  it('render a card with an image, brand, model and price', () => {
+    render(<ProductListing brand="Nike" model="Air Force 1" price="79,99€" />)
+    expect(screen.getByRole('img')).toBeInTheDocument()
+    expect(screen.getByText('Nike')).toBeInTheDocument()
+    expect(screen.getByText('Air Force 1')).toBeInTheDocument()
+    expect(screen.getByText('79,99€')).toBeInTheDocument()
+  })
+  it('should display an overlay with available sizes on hover', () => {
```

Comment:
Maybe you should think about whether it should be on `hover` as you want to build it mobile first.

Reply:
Make sense 😅